tests/cuda_interop/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+
+if(SYCL_CTS_ENABLE_CUDA_INTEROP_TESTS)
+    file(GLOB test_cases_list *.cpp)
```

Comment:
There are CUDA files in the test. How tests/cuda_interop/cuda_interop_device_function.cu is compiled?

Reply:
I have updated the CMake to build `.cu` files in the same way .cpp files are built. 
Using dpcpp as an example the compiler call should be the same as with regular sycl. The compiler should recognise the fact it contains cuda offloading via whe `.cu` prefix